### DISCUSSION TITLE
Is this still maintained? Do you need help?

### DISCUSSION THREAD

**Discussion by dylandhall** [anasfik/openai#179] [General]
I opened a PR a week ago with an improvement someone else had requested, and while I'm using the fork locally it would be nice to continue using this package. There's another PR in there as well - does the original maintainer need help reviewing submissions?

This is a really useful library, would hate to see it go unmaintained. I wouldn't want to open a fork of it because I can't guarantee I can spend time maintaining it but perhaps there could be some assistance in reviewing PRs or similar?

**Comment by franboladoruiz** [comment]
I ended up migrating to this package, it seems to be better maintained:
- https://pub.dev/packages/openai_dart

**Comment by dylandhall** [comment]
oh nice, thank you!
